help/en/releasenotes/current-draft-note.shtml

```diff
@@ -126,7 +126,14 @@
 
         <h4><a href="http://openlcb.org">OpenLCB</a> / LCC</h4>
             <ul>
-                <li></li>
+                <li>There's a new storage method for event names.
+                    Previously, these were stored in the IdTag table, which
+                    be confusing.
```

Comment:
No need to change.
I'm not an English native speaker so I might be wrong, but it seems to me that the word "might" is missing before "be confusing".

Previously, these were stored in the IdTag table, which might be confusing.